data/styles/default/include/Icons.mapcss

```diff
@@ -201,6 +201,13 @@ node|z19-[man_made=water_tap],
 node|z19-[man_made=water_well],
 {icon-image: drinking-water-m.svg;}
 
+node|z16-[man_made=water_tap][drinking_water=not],
+node|z16-[man_made=water_well][drinking_water=not],
+{icon-image: none;}
+node|z18-[man_made=water_tap][drinking_water=not],
+node|z18-[man_made=water_well][drinking_water=not],
+{icon-image: drinking-water-no-m.svg;}
```

Comment:
On z18 a `drinking-water-s.svg` (small) is used for drinkables sources, so we need a small version of a non-drinkable icon too.

Also need to change Icons.mapcss for the Outdoors accordingly (water sources are displayed earlier there, drinkable are z12-, we can make non-drinkable z14-)

Reply:
Added small non-drinking-water icon 

Added to outdoors